BP-TPSE-Projektgruppe-80/NaPyTau
# Add Import demo

### Summary

This adds a demo in the headless module to show off reading a napatau style dataset.

---

### PR checklist

- [x] I have written tests that fail without my changes. _Or I did not add them on purpose._
- [x] I have included instructions on how to test this, assuming they are not already apparent from the linked issue.

---

### Depends on


- [x] Depends on #17 
- [x] Depends on #32 

## How to test

- Run the application in headless mode, providing a setup_files_directory_path pointing to a valid directory containing setup files in napatau format.
- Check that the printed out values match the provided setup files.
